src/components/time-display/time-display.js

```diff
@@ -48,10 +48,10 @@ class TimeDisplay extends Component {
    * @returns {React$Element} - component
    * @memberof TimeDisplay
    */
-  render(props: any): React$Element<any> {
+  render({currentTime, duration, format = 'current / total'}: any): React$Element<any> {
```

Comment:
can total be 0 ?

Reply:
its just format, any way the logic is the same as before not touched it , just syntax difference